### PR TITLE
Website

### PR DESCRIPTION
**Pull request by mj-e-boc**
- Implement desktop version of portfolio

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -7,9 +7,9 @@ body {
 .frontpage {
```
Comment (danmainah) [inline styles.css:7]: - [ ] Kindly check on your design the header section and footer don't match the Figma design:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

 
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: index.html [hunk 2]
```diff
@@ -100,19 +109,19 @@ <h2>Calculator</h2>
             <li class="skillitem">css</li>
             <li class="skillitem">javascript</li>
           </ul>
-          <button class="projectbutton" type="button">See Project</button>
+          <button class="project1button" type="button">See Project</button>
```
Comment (danmainah) [inline index.html:112]: - [ ] Kindly ensure you implement the hover and press effect in your buttons design:

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @mj-e-boc , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (mj-e-boc) [reply index.html:112]: hey, please ignore my submissions, and help me cancel further ones on this activity I was working on day 3 project not day 4,

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @mj-e-boc ✋ 



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (mj-e-boc) [reply index.html:112]: figured it out , don't worry, thanks for the help

**Pull request merged**